go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	github.com/microsoftgraph/msgraph-sdk-go v0.47.0
 	github.com/nutanix-cloud-native/prism-go-client v0.2.1-0.20220804130801-c8a253627c64
 	github.com/onsi/gomega v1.28.1
-	github.com/openshift/api v0.0.0-20231019121755-0c5924392281
+	github.com/openshift/api v0.0.0-20231115210901-4c4a0a24f2fc
```

Comment:
This seems to be pointing to the 4.14 backport? https://github.com/openshift/api/commit/4c4a0a24f2fc
Because of that the build is failing with 
```
# github.com/openshift/installer/pkg/types/validation
pkg/types/validation/installconfig.go:166:98: undefined: configv1.ClusterVersionCapabilityOperatorLifecycleManager
```